docs/pages/setup/operations/upgrading.mdx

```diff
@@ -64,16 +71,18 @@ When upgrading a single Teleport cluster:
 >
   If several Auth Service instances are running in the High Availability configuration
   (for example, in an AWS Auto Scaling group), you must shrink the group to
-  **just one Auth Service** before performing an upgrade. 
-  
-  While Teleport will attempt to perform any necessary migrations, we recommend users create a backup of their backend before upgrading the Auth Server as a
-  precaution. This allows for a safe rollback in case the migration itself fails.
+  **just one Auth Service** before performing an upgrade.
+
+While Teleport will attempt to perform any necessary migrations, we recommend users create a backup of their backend before upgrading the Auth Server as a
```

Comment:
```suggestion
While Teleport will attempt to perform any necessary migrations, we strongly recommend users create a backup of their backend before upgrading the Auth Server as a
```

Reply:
Meh, I think I'll leave it as is. It doesn't read much differently, and I don't expect anyone to change their behavior based on whether we recommend or strongly recommend something.